tools/gulp/tasks/docs.ts

```diff
@@ -111,7 +111,7 @@ task('minified-api-docs', ['api-docs'], () => {
 
 /** Copies example sources to be used as plunker assets for the docs site. */
 task('plunker-example-assets', () => {
-  src(path.join(SOURCE_ROOT, 'examples', '**/*'))
+  src(path.join(SOURCE_ROOT, 'material-examples', '**/*'))
```

Comment:
While you're in here, can you remove the extra `plunker/` that's being written to the docs content?